srcjs/output_binding_image.js

```diff
@@ -525,12 +525,22 @@ imageutils.initCoordmap = function($el, coordmap) {
         }
         if (clip)
           return;
-      }
-      if (clip && !coordmap.isInPanelCss(offset_css)) return;
 
+        const coords_px = coordmap.scaleCssToImg(offset_css);
```

Comment:
offset_px

Reply:
scratch that... use the left hand side values for all vars in this function.  ex keep `coords_css` and make `offset_css` `coords_css`